app/lib/bk/compat/parsers/bitbucket/parallel.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require_relative '../../pipeline/step'
-
```

Comment:
This file still uses some classes so it should include the dependencies explicitly:

```suggestion
require_relative '../../models/steps/group'
require_relative '../../models/steps/wait'

```

Reply:
Added back in 👍